Name and Link Consistency (#194)
returntocorp/semgrep-docs: docs/ignoring-findings.md

    The space (` `) before `nosemgrep` is required for Semgrep to detect this annotation.

# File paths

!!! note
    Semgrep will respect the presence of a `.gitignore` in the project and will not scan any paths present in that file. This behavior can be disabled by using the flag `--no-git-ignore`.

> For Semgrep CI users, see the [Ignoring files](semgrep-ci.md#ignoring-files) documentation.

Reviewer: ```suggestion
Semgrep CI users can also [use a `.semgrepignore` file](semgrep-ci.md#ignoring-files).
```

> If you're directly running the Semgrep command line tool, use the `--exclude <pattern>` flag.

Reviewer: ```suggestion
If you're directly running the Semgrep command line tool, use the `--exclude <pattern>` flag.

```
